assignments/prototypes.js

```diff
@@ -33,6 +54,23 @@
   * should inherit takeDamage() from CharacterStats
 */
  
+function Humanoid ({createdAt, dimensions, healthPoints, name, team, weapons, language}){
+  CharacterStats.call(this);
+  this.createdAt = createdAt;
```

Comment:
When you are doing a `.call`, you inherit are inheriting the properties of the function your are binding.  So when you do the call, the attributes that are passed in are passed up to the definition that is already called.

It should look like this:

```js
function Humanoid(humanoidAttrs) {
     CharacterStats.call(this,  humanoidAttrs);
     this.team = humanoidAttrs.team;
     this.weapons = humanoidAttrs.weapons;
     this.language = humanoidAttrs.language;
}
```